bin/tomqueued

```diff
@@ -3,31 +3,51 @@
 require "./config/environment"
 require 'optparse'
 
-@worker_options = {
-  :min_priority => ENV['MIN_PRIORITY'],
-  :max_priority => ENV['MAX_PRIORITY'],
-  :queues => (ENV['QUEUES'] || ENV['QUEUE'] || '').split(','),
-  :quiet => ENV['QUIET']
+options = {
+  workers: 1,
+  deferred_scheduler: false,
+  executable_path: "./script/tom_queue_worker_boot.rb",
```

Comment:
I've added this in as a default otherwise we'd need to remember to add it in each time we run `bundle exec tomqueued`. Happy to remove it though if you prefer.

Reply:
eek yeah - I'd probably default it to `config/tomqueue_config.rb` ?